### ISSUE TITLE
Ratings nag message

### ISSUE THREAD

**Issue by ghost** [bitfolk/enoch#13]
Every time `enoch` says a quote in a channel, it could go through the channel list looking for people who are able to rate the quote.

> (You're able to rate a quote if:
> - Your nickname is registered and you're identified to it;
> - You didn't add the quote, and;
> - You haven't already rated it)

For each such person it finds, it could send them a private message asking them if they want to rate the quote 1-10. A response simply containing a number between 1 and 10 would be enough to set the rating. In this way people would be encouraged to rate, and it's less cumbersome than having to type out a full "!ratequote …" command. It also avoids you having to guess if you are able to rate or have not already rated a given quote.

There could be other responses ("shut up", "die", "bugger off", "no", …?) which would disable this feature for this person.

If this was thought to be too irritating for people who have no interest in the quotes then it could also be restricted to people who have ever rated a quote for that channel. That way, if you've never rated a quote before, `enoch` will never ask you to do so.

Certainly there would have to be a way to disable the feature on a per-person basis. Perhaps it would be best starting off disabled for everyone?

**Comment by domcleal** [comment]
Could it be configurable per-channel?  I don't think we'd want this spamming away in #surrey at least.

**Comment by pidg** [comment]
I would not use this feature

**Comment by bitfolk** [comment]
I suppose it could be enabled per-channel. Although I was proposing that it would talk to people in private message, so at no point would it be spamming away in any channel.

**Comment by domcleal** [comment]
Indeed, it's more that it would annoy the people joined the channel even if it isn't filling the channel itself up.  Opt-in would be better than opt-out in that situation.

**Comment by bitfolk** [comment]
Even if it only ever messaged people who have actually rated quotes before (13 people), and they could stop it happening permanently by typing "no" in response?
